tests/sentry/workflow_engine/test_base.py

```diff
@@ -66,9 +69,13 @@ def create_detector_and_workflow(
         self,
         name_prefix="test",
         workflow_triggers: DataConditionGroup | None = None,
-        detector_type: DetectorType | str = "TestDetector",
+        detector_type: DetectorType | str = DetectorType.METRIC_ALERT_FIRE,
         **kwargs,
     ) -> tuple[Workflow, Detector, DetectorWorkflow, DataConditionGroup]:
+        """
+        Create a Worfkllow, Detector, DetectorWorkflow, and DataConditionGroup for testing.
```

Comment:
I wouldn't normally nitpick typos like this, but since some tests are failing anyway I thought I would point out this docstring 😅